src/Native/CMakeLists.txt

```diff
@@ -43,7 +43,12 @@ elseif (CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l)
     add_definitions(-D_FILE_OFFSET_BITS=64)
     # Because we don't use CMAKE_C_COMPILER/CMAKE_CXX_COMPILER to use clang
     # we have to set the triple by adding a compiler argument
+  if(ARM_SOFTFP)
+    add_compile_options(-target armv7-linux-gnueabi)
```

Comment:
Again, what is the difference between `armv7-linux-gnueabi` and `arm-linux-gnueabi`?